main2.py

```diff
@@ -0,0 +1,42 @@
+import logging
+
+
+# Import database logic
+# import query_database
+# GBIF API functionality
+import GBIF_functions
+# GeoCASe API functionality
+import GeoCASe_functions
+import query_database
+
+logging.basicConfig(format='%(asctime)s - %(message)s', level=logging.INFO)
+
+
+def process_countries():
+    logging.info('\nReceiving countries datasets data from GBIF...')
```

Comment:
The '\n' should be necessary to enforce a new line. Are you sure this is needed now you use the logger?

Reply:
Correct assumption, removed the '\n' elements